### PR TITLE
Build  73.0.3683.103-1 for Ubuntu Cosmic

### LINKED ISSUE #6: clang-7: error: unknown argument: '-fsplit-lto-unit'
```
clang-7: error: unknown argument: '-fsplit-lto-unit'
ninja: build stopped: subcommand failed.
make[1]: *** [debian/rules:140: override_dh_auto_build-arch] Error 1
make[1]: Leaving directory '/usr/src/sources/2019/browser/ung73/build/src'
make: *** [debian/rules:114: binary] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
```

I've changed it to clang-8, then the compiling continues to run.

### PR DESCRIPTION
**Pull request by riyad**
I have a working build for ` 73.0.3683.103-1` for `ubuntu_cosmic`.

Steps:
- [x] merge `debian_buster` into `ubuntu_cosmic`
- [x] use LLVM-8 to prevent `clang++-7: error: unknown argument: '-fsplit-lto-unit'` (see https://github.com/ungoogled-software/ungoogled-chromium-debian/commit/75f3e55921e911955805d130c96d74a510e02cd7; fixes #6) 
- [x] integrate [this patch](https://aur.archlinux.org/cgit/aur.git/tree/chromium-media-fix-build-with-libstdc++.patch?h=chromium-vaapi) to prevent  `base/optional.h:348:61: error: no member named 'value' in 'std::is_constructible<media::learning::Value, const base::Optional<media::learning::Value> &>'` (thanks to @Eloston fixing this in https://github.com/ungoogled-software/ungoogled-chromium-debian/pull/8/commits/5beef9aca18b5976302f1bc39bf9dbcc3e543cf0)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,12 +36,26 @@ mkdir -p build/src
...
+
 cp -r ungoogled-chromium-debian/debian build/src/
 cd build/src
 
 # Final setup steps for debian/ directory
 ./debian/rules setup-debian
 
+# Add packages for LLVM 8
+# The easiest way to do this is to use the APT repo from apt.llvm.org
+# 1. Add this line to your /etc/apt/sources.list: deb http://apt.llvm.org/buster/ llvm-toolchain-buster-8 main
```
Comment (Eloston) [inline README.md]: This should probably be `cosmic` instead of `buster`

**Review by Eloston — CHANGES_REQUESTED** [review]

**Comment by Eloston** [comment]
Also, I'll look to merging that last patch into `debian_buster` since it looks like it affects anyone that uses `libstdc++`

**Comment by Eloston** [comment]
Just pushed the patch: 5beef9aca18b5976302f1bc39bf9dbcc3e543cf0

**Comment by riyad** [comment]
@Eloston awesome, thank you. :+1: 

I rebased the PR on top of your last commits ... there's basically nothing left besides just merging `debian_buster`. I'll do a rebuild with the updated set of patches. If it succeeds ...

- this PR can be merged
- you can tag the `73.0.3683.103-1.cosmic1` release
- I'll create a PR for the binary release

File: README.md (hunk 1 above)
Comment (riyad) [reply README.md]: done

**Comment by riyad** [comment]
@Eloston from my side this looks good.

File: README.md (hunk 1 above)
Comment (riyad) [reply README.md]: see https://github.com/ungoogled-software/ungoogled-chromium-debian/pull/8/commits/ac16b1edaceb9082caed7d3ee6b30806440de35c

**Review by Eloston — APPROVED** [review]
LGTM. Thanks :+1:

**Pull request merged**